tests/data/sample_file.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0"?>
+<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Publishing DTD v1.2 20190208//EN" "JATS-journalpublishing1.dtd">

```

Comment:
Is there a reason to use v1.2 while v1.3 is the current version?

Reply:
This file was taken directly on JATS website. The only thing changed were mainly the metadata inside because it was an real article. I am not sure this version has an impact on the parsing itself.

Reply:
Following our discussion, I have checked what is the JATS version used in PMC. I have used the article put as example for their OA service [here](https://www.ncbi.nlm.nih.gov/pmc/utils/oa/oa.fcgi?id=PMC5334499). The version is v1.0, which is lower than the one currently used in the parser of the PR. There might then be some issues.

```xml
<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD v1.0 20120330//EN" "JATS-archivearticle1.dtd">
```

Reply:
Good point, @pafonta ! I just checked with the data we have for now. It seems that several versions are used depending on the article. Check some stats I found here:

| Version | Number of Articles |
| ---- | ---- |
| Journal Archiving and Interchange DTD v1.0 | 826'545 |
| Journal Archiving and Interchange DTD v2.3 | 73'875 |
| Journal Archiving and Interchange DTD v3.0 | 1 |

Reply:
After checking manually, it seems we are able to extract all informations needed (`title`, `authors`, `abstract` and `paragraphs` - also `figure` and `table`) no matter the version of the schema.

Reply:
Thank you @EmilieDel for checking.

JATS v2.3 and 3.0 do not exist, so I guess they are incorrect data.